decentraland/unity-renderer
# fix: [BiW Editor] Disable objects culling

## What this PR includes?

### Fixes #624 
- [x] Disable objects culling during the edition.

### Fixes #619 
- [x] Sometimes, when you enter to edit a Land that has an associated Project, the assets (3D Models) do not load until you add another entity to the scene.

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/BiW-Disable-objects-culling&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=80,154
2. Press 'K' to open the Builder In World editor.
3. Notice all the entities are visible.
4. Exit from the editor.
5. Enter back into the editor.
6. Notice all the entities are visible.
7. Repeat the process several times.

## Annotations
Finally I had not have to use the new implemented class `SettingsPanelDataStore` for this issue, but I would like to keep it in the project because I think it is something useful that we're going to use in the future. This data store class let us to change any setting control from any place of our code without the necessity to go and click fisically in the control inside the SettingsPanelHUD.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
